samples/app-identity-linking-with-sso/csharp/README.md

```diff
@@ -43,6 +49,8 @@ Messaging Extention mapping
 - ME Link unfurling
 
   ![ME Link unfurling](IdentityLinkingWithSSO/Images/MELinkUnfurlingCard.png)
+
+  - On Click of `Disconnect from facebook` or `Disconnect from google`. User's facebook/google data will be removed from the data base.
```

Comment:
Database
Also, are we using any database or just saving it in memory.
So we can add a extensibility note like, you can use any memory like DB or in memory or in-file.